### DISCUSSION TITLE
Voice prompts for every bend in the road

### DISCUSSION THREAD

**Discussion by osmannetje** [osmandapp/OsmAnd#18128] [Q&A]
I am a very satisfied Osmand user. But now that I want to use Osmand as an offline navigation app in the car, I encounter the following problem. At almost every bend on the set route, speech and images indicate that I have to turn right or left after a few meters, depending on the bend. And this a number of times for each bend. There are no side roads in the area. Very exhausting! I have now only activated in the profile that the exits must be indicated, all other instructions have been disabled. I am using Android 10 and Osmand 4.5.10. It would be nice if someone has a solution for this problem.

**Comment by tommycrock** [comment]
Hi @osmannetje 
Not my area of expertise but it will probably help if you say how you are setting the navigation up. Are you just pointing at a place in osmand and clicking navigate or are you trying to follow a gpx for example?

**Reply by osmannetje** [reply]
Thanks for your comment. You're right, it's better to report that. I started the navigation by clicking on a point on the map and then starting the navigation. I have to make a correction: it doesn't happen at every turn, but at **almost** every turn.

**Reply by DmitryAlexei** [reply]
@osmannetje 
Please specify the start and end points so that we can reproduce this issue

**Reply by osmannetje** [reply]
Thank you for your response. It doesn't seem very useful to me to give the exact departure and arrival coordinates. He makes the same mistakes on all the routes I want to ride, both in Denmark and in the Netherlands. On all types of roads.

**Comment by Sonwon1** [comment]
Are you creating the route with OsmAnd or are you importing the route you created from a GPX file?

**Comment by osmannetje** [comment]
I am creating the route with Osmand and not with a GPX file. It is an effect that I have not read anywhere yet. I hope someone can help me on my way.

Verzonden met [Proton Mail](https://proton.me/) beveiligde e-mail.

------- Original Message -------
Op zondag 10 september 2023 om 11:09 schreef Sonwon1 ***@***.***>:

**Comment by Sonwon1** [comment]
Yeah, I have only experienced that with GPX files. I am going on a 16 day tour and will post here if that changes.

**Reply by sjvudp** [reply]
A GPX track just has a set of points, no roads. So I guess when the path makes a turn, a turn is announced, wherever you are.

**Comment by osmannetje** [comment]
Thank you for your response. I hope you come back with a solution.

**Comment by osmannetje** [comment]
The problem is solved. I found the solution in a Google forum. The problem was caused by the automatic map updates, which apparently slightly disrupted the location of roads on the map. Removed the map, downloaded it again and turned off automatic updates. Everything works properly again. Thank you folks for the responses.

**Comment by pedjas** [comment]
It seems I have the same issue. I managed to record it: https://www.youtube.com/watch?v=n_WPsNXp390

I just choose destination and let OsmAnd callcualte route. This happens on every route, all over the map.

I will try this map re-download suggestion to see if it would help.

**Comment by osmannetje** [comment]
Ok, it helped in my situation. Good luck.
